avaframe/com1DFA/plotTrackParticles.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
```

Comment:
I think it is not a good idea to have the plot show (i.e. popup) by default. I think the default should be to plot to file and only show the plot if it was specifically requested